app/components/page-components/Chatbot/Commands/ChatbotDefaultCommands.vue.ts

```diff
@@ -0,0 +1,49 @@
+import ChatbotBase from 'components/page-components/Chatbot/ChatbotBase.vue';
+import { Component } from 'vue-property-decorator';
+import {
+  DafaultCommandsResponse,
+  ChatbotAPIPostResponse
+} from 'services/chatbot/chatbot-interfaces';
+
+
+@Component({})
+export default class ChatbotDefaultCommands extends ChatbotBase {
+  commandSlugs: DafaultCommandsResponse = null;
+
+  mounted() {
+    //
+    // get list of user's default commands
+    //
+    this.fetchCommands();
+  }
+
+  fetchCommands() {
+    // fetch default commands
+    this.chatbotApiService
+      .fetchDefaultCommands()
+      .then((response: DafaultCommandsResponse) => {
+        console.log(response);
+        this.commandSlugs = response;
+      })
+      .catch(err => {
+        alert('Error fetching default commands');
```

Comment:
Eventually we probably want better error handling than this.  This is probably fine for now.

Reply:
You can also look at `vue-toasted` plugin that @gettinToasty added for error notifications